libraries/apollo-normalized-cache-api-incubating/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/api/RecordMerger.kt

```diff
@@ -111,41 +111,95 @@ private object ConnectionFieldMerger : FieldRecordMerger.FieldMerger {
     return if (incomingBeforeArgument == null && incomingAfterArgument == null) {
```

Comment:
Add the link to the [Relay issue](https://github.com/facebook/relay/issues/3850) as a comment somewhere here?